### ISSUE TITLE
Create a model to detect hand-drawn "SOS"

### ISSUE THREAD

**Issue by krook** [Code-and-Response/DroneAid#7]
**Is your feature request related to a problem? Please describe.**
We assume that the person in need will have a kit with the printed symbols available. We should improve the system to demonstrate how a person could hand-recreate the symbols, and in turn, make the recognition more sensitive to those symbols.

**Comment by krook** [comment]
Very timely: https://www.cnn.com/2019/10/17/world/missing-australian-woman-sos-rescued-trnd/

**Comment by anushkrishnav** [comment]
I can work on this

**Comment by krook** [comment]
Thanks @anushkrishnav

**Comment by sarrah-basta** [comment]
Is this issue resolved, as I can see it open and unassigned? My proposal for the resolving it is:  If we already have the models for our symbols from the kit ready, and a hand-written SoS is not one of them, it could be a good idea to instead use a pretrained model for handwritten letter recognition and integrate it with our present models.

**Comment by krook** [comment]
Hi @sarrah-basta. This hasn't been worked on yet. Please feel free to take a shot. Thank you! And I agree, if we can reuse a model that would be ideal. Maybe the [Model Asset Exchange](https://developer.ibm.com/exchanges/models/) has something to build upon.

**Comment by bhavyagoel** [comment]
Hi, @krook hope you are doing well!
As the issue is opened and unassigned, I can work on this. I would propose to simply determine the alphabets written on the ground, as it could be possible that someone in need might write some other information as well, like for reference "INJURED" or something else. 
And based on the determined text, we can classify it. 
For this, we can use a pre-trained model, and as mentioned by you [Model Asset Exchange](https://developer.ibm.com/exchanges/models/) seems a good choice, we can also take help from [mediapie](https://mediapipe.dev/) for detecting if the human is present their or not, and in what state.

**Comment by sarrah-basta** [comment]
@krook @bhavyagoel yes as both of you'll have mentioned, we can use [Optical character Recognition](https://developer.ibm.com/exchanges/models/all/max-ocr/) from [Model Asset Exchange](https://developer.ibm.com/exchanges/models/all/) to detect the text. It can be further classified using either manually or using a classification model such as Naive Bayes classifier or the like. And adding the [Face Detection Model](https://google.github.io/mediapipe/solutions/face_detection) from [MediaPipe](https://mediapipe.dev/) as mentioned would also be a huge plus.

**Comment by krook** [comment]
Thanks for your interest everyone. So since @sarrah-basta replied first to the latest request, why don't you take the first pass at it? If you need any feedback or review of the proposed approach, then you can tag @bhavyagoel and I. Sound like a plan?

**Comment by sarrah-basta** [comment]
Yep sure, I'll start by finalising the models among ones we were proposing and try looking at our codebase to understand how it can be integrated. Thanks

> Thanks for your interest everyone. So since @sarrah-basta replied first to the latest request, why don't you take the first pass at it? If you need any feedback or review of the proposed approach, then you can tag @bhavyagoel and I. Sound like a plan?
